crates/runtime/src/dataaccelerator/partitioned_duckdb.rs

```diff
@@ -358,6 +361,13 @@ impl PartitionCreator for DuckDBPartitionCreator {
         );
         Ok(partitions)
     }
+
+    fn supports_filters_pushdown(
+        &self,
+        filters: &[&Expr],
+    ) -> Result<Vec<TableProviderFilterPushDown>, DataFusionError> {
+        Ok(vec![TableProviderFilterPushDown::Exact; filters.len()])
```

Comment:
We should first run the `Expr` through `expr_to_sql` to make sure we can properly unparse it: https://docs.rs/datafusion/latest/datafusion/sql/unparser/fn.expr_to_sql.html

If we can, then its exact. If we can't then its unsupported.